platform/glfw/ny_route.hpp

```diff
@@ -0,0 +1,103 @@
+#include <string>
+
+namespace mbgl {
+namespace platform {
+namespace glfw {
+
+constexpr const char* route = "{\
```

Comment:
Can we please use a [raw string literal](http://en.cppreference.com/w/cpp/language/string_literal) like we do in the [shader code](https://github.com/mapbox/mapbox-gl-native/blob/master/src/mbgl/shaders/fill.cpp#L9)?